ludwig/utils/data_utils.py

```diff
@@ -205,22 +221,28 @@ def read_xsv(data_fp, df_lib=PANDAS_DF, separator=",", header=0, nrows=None, ski
 
 @DeveloperAPI
 @spread
-def read_json(data_fp, df_lib, normalize=False):
+def read_json(data_fp, df_lib, normalize=False, **kwargs):
+    # Chunking is not supported for JSON files unless lines=True:
```

Comment:
Do we want to add logic to support chunking with `lines=True` here?

Reply:
My impression is that that is the purpose of `read_jsonl` , so I'd say no